astropy/io/votable/tests/table_test.py

```diff
@@ -23,7 +24,11 @@
 
 def test_table(tmpdir):
     # Read the VOTABLE
-    votable = parse(get_pkg_data_filename('data/regression.xml'))
+    with warnings.catch_warnings():
+        # https://github.com/astropy/astropy/issues/13341
+        warnings.filterwarnings(
+            'ignore', message='overflow encountered in cast', category=RuntimeWarning)
```

Comment:
Just as a note:  Another option is to use `with np.errstate(over="ignore")` or `@np.errstate(over="ignore")` as a decorator.

Reply:
That looks cleaner. I'll refactor this when I get the chance. Thanks!